src/common/_modules/redis/locals.tf

```
@@ -7,3 +7,20 @@ locals {
     }
   }
 }
+
+###Italy North
+locals {
+  prefix    = "io"
+  env_short = "p"
+  # domain          = "redis"
+  app_name        = "redis"
+  instance_number = "01"
+  itn_environment = {
+    prefix    = local.prefix
+    env_short = local.env_short
+    location  = var.location
+    app_name  = local.app_name
+    # domain          = local.domain
```

I think it shall be `auth` cc @BurnedMarshal 
```suggestion
    domain          = "auth"
```